localflavor/gr/forms.py

```diff
@@ -60,3 +61,56 @@ def clean(self, value):
         if mod != check:
             raise ValidationError(self.error_messages['invalid'])
         return val
+
+
+class GRSocialSecurityNumberCodeField(Field):
```

Comment:
It would be better use `RegexField` instead of `Field`. You would then set the regex on the call to super.

Reply:
Ok